### PR TITLE
Fix mailing list link in the dashboard

### PR DESCRIPTION
**Pull request by mcmikashi**
We can now use the link on the dashboard.

### CODE REVIEW

**Comment by Shriukan33** [comment]
the only available alternative to horizontal scrolling an HTML table would be to use [datatables](https://datatables.net/), it uses jQuery to do so. 
In the example given you click a name to see the details that are normally displayed in columns. 

What do you think  @JeffAbrahamson  ?

File: transport_nantes/mailing_list/templates/mailing_list/mailing_list_list_view.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+			<th scope="col">État</th>
+			<th scope="col">Nombre d'inscrits</th>
+		</tr>
+		</thead>
+		<tbody>
+		{% for mailing_list,count_subscriber in mailing_lists %}
+		<tr>
+			<td>{{ mailing_list.mailing_list_name }}</td>
+			<td>{{ mailing_list.contact_frequency_weeks }}</td>
+			<td>{% if mailing_list.list_active %} Activer {% else %} Désactiver {% endif %}</td>
```
Comment (Shriukan33) [inline transport_nantes/mailing_list/templates/mailing_list/mailing_list_list_view.html:20]: "Activé" if you're talking about its status, "Activer" if you mean to place a button to activate the list. It's an action verb.

File: transport_nantes/mailing_list/templates/mailing_list/mailing_list_list_view.html [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+		  <th scope="col">Nom</th>
+		  <th scope="col">État</th>
+		  <th scope="col">Nombre d'inscrits</th>
+	  </tr>
+	  </thead>
+	  <tbody>
+	  {% for petitions_lists,count_subscriber in petitions_lists %}
+	  <tr>
+		  <td>{{ petitions_lists.mailing_list_name }}</td>
+		  <td>{% if petitions_lists.list_active %} Activer {% else %} Désactiver {% endif %}</td>
```
Comment (Shriukan33) [inline transport_nantes/mailing_list/templates/mailing_list/mailing_list_list_view.html:39]: Same thing about activer

**Review by Shriukan33 — CHANGES_REQUESTED** [review]

**Comment by JeffAbrahamson** [comment]
> the only available alternative to horizontal scrolling an HTML table would be (afaik) to use [datatables](https://datatables.net/), it uses jQuery to do so. In the example given you click a name to see the details that are normally displayed in columns. Another example : https://datatables.net/extensions/fixedheader/examples/integration/responsive-bootstrap.html What do you think @JeffAbrahamson ?

"only" ?  That's a very strong constraint...

There's [overflow-x:auto](https://www.w3schools.com/howto/howto_css_table_responsive.asp).  And [this css trick](https://css-tricks.com/responsive-data-tables/).  And bootstrap's [.table-responsive](.table-responsive).  Just to pull out some of the first DDG hits.

**Comment by Shriukan33** [comment]
> > the only available alternative to horizontal scrolling an HTML table would be (afaik) to use [datatables](https://datatables.net/), it uses jQuery to do so. In the example given you click a name to see the details that are normally displayed in columns. Another example : https://datatables.net/extensions/fixedheader/examples/integration/responsive-bootstrap.html What do you think @JeffAbrahamson ?
> 
> "only" ? That's a very strong constraint...
> 
> There's [overflow-x:auto](https://www.w3schools.com/howto/howto_css_table_responsive.asp). And [this css trick](https://css-tricks.com/responsive-data-tables/). And bootstrap's [.table-responsive](.table-responsive). Just to pull out some of the first DDG hits.

Haha yeah, maybe shouldn't have use a strong word like this, more "the only I'm aware of that comes to my mind" :laughing:

**Comment by mcmikashi** [comment]
For the table I see this [link](https://examples.bootstrap-table.com/index.html#welcome.html)

File: transport_nantes/mailing_list/templates/mailing_list/mailing_list_list_view.html (hunk 1 above)
Comment (JeffAbrahamson) [reply transport_nantes/mailing_list/templates/mailing_list/mailing_list_list_view.html]: Isn't it "active" -- une liste active ou pas active ?

**Comment by JeffAbrahamson** [comment]
The phrase "Fréquence des contacts par semaine(s)" is wrong.  Did you look a the data ?  A trimestrielle liste doesn't send 12 messages per week.  Indeed, the number is an integer, it couldn't very well be 1/12 for a trimestrielle list.

This is an admin dashboard.  Perhaps just say "Fréquence de contacte (en semaines)" and be done with it?

**Comment by JeffAbrahamson** [comment]
It's still not responsive (I sometimes see a horizontal scrollbar, but not always), but it's admin, so LGTM apart the frequency label change.

image

**Comment by mcmikashi** [comment]
I made the change for the the frequency label.

**Comment by JeffAbrahamson** [comment]
LGTM.

**Pull request merged**